iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -250,4 +259,43 @@ private static PartitionSpec spec(Schema schema, Properties properties,
       return PartitionSpec.unpartitioned();
     }
   }
+
+  @Override
+  public void commitInsertTable(org.apache.hadoop.hive.metastore.api.Table table, boolean overwrite)
```

Comment:
In case of multi-table insert this method is called for every table?

Reply:
yes